hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestRepairsCommand.java

```diff
@@ -95,8 +95,9 @@ public void testAddPartitionMetaWithDryRun() throws IOException {
         .toArray(String[][]::new);
     String expected = HoodiePrintHelper.print(new String[] {HoodieTableHeaderFields.HEADER_PARTITION_PATH,
         HoodieTableHeaderFields.HEADER_METADATA_PRESENT, HoodieTableHeaderFields.HEADER_REPAIR_ACTION}, rows);
-
-    assertEquals(expected, cr.getResult().toString());
+    expected = removeNonWordAndStripSpace(expected);
```

Comment:
Actually I added this entire class TestRepairsCommand by mistake. 0.5.3 doesn't pull in HUDI-704 and hence have I have deleted this file in my latest commit.